packages/vite/src/node/plugins/css.ts

```diff
@@ -1504,8 +1504,16 @@ function loadPreprocessor(
     return (loadedPreprocessors[lang] = _require(resolved))
   } catch (e) {
     if (e.code === 'MODULE_NOT_FOUND') {
+      const preprocessorPackage = {
+        [PreprocessLang.scss]: 'sass',
+        [PreprocessLang.sass]: 'sass',
+        [PreprocessLang.less]: 'less',
+        [PreprocessLang.styl]: 'stylus',
+        [PreprocessLang.stylus]: 'stylus',
+        [PostCssDialectLang.sss]: 'sugarss',
+      }
       throw new Error(
-        `Preprocessor dependency "${lang}" not found. Did you install it?`,
+        `Preprocessor dependency "${lang}" not found. Did you install it? Try \`npm i -D ${preprocessorPackage[lang]}\`.`,
```

Comment:
I think this is a nice QoL, but we may need to handle different package managers too, perhaps deriving from `process.env.npm_config_user_agent` is good enough to prevent fs access. ([example](https://github.com/vitejs/vite/blob/13ac37df8e135fe6b4990fac6f149a36a19e5409/packages/create-vite/src/index.ts#L312))

I also don't think we need the `preprocessorPackage` map, we can use the `lang` variable directly as the package name.

Reply:
Hi, @bluwy Thanks for your response!

1. That sounds like a fair solution! I will try to implement it.
2. However, `lang` can be a string such as `scss` or `sass`. There are different packages available and we want to use `sass`. Did I miss anything?

Reply:
I don't think it could be either `scss` or `sass`. It's called here:

https://github.com/vitejs/vite/blob/13ac37df8e135fe6b4990fac6f149a36a19e5409/packages/vite/src/node/plugins/css.ts#L1564

So it should be always `sass`.

Reply:
Got it! Thanks for your explanation.